coursera_autograder/commands/upload.py

```diff
@@ -422,11 +419,11 @@ def setup_registration_parser(parser):
         '--grader-memory-limit',
         type=int,
         choices=list(range(2048, 16384, 1024)),
-        default = 2048,
+        default = 4096,
         help='Amount of memory your grader is allocated when grading '
              'submissions. Amount of memory you can request for 1 CPU is '
              'between 2048 MB to 8192 MB. For 2 CPUs is between 4096 MB to 16384 MB.'
```

Comment:
Do we also need to change the amount of memory available for 2 CPUs?
```suggestion
             'between 4096 MB to 8192 MB. For 2 CPUs is between 4096 MB to 16384 MB.'
```